src/components/OpenSeadragonViewer.js

```diff
@@ -105,12 +103,8 @@ export class OpenSeadragonViewer extends Component {
         this.osdCanvasOverlay.clear();
         this.osdCanvasOverlay.resize();
         this.osdCanvasOverlay.canvasUpdate(() => {
-          if (highlightsUpdated) {
-            this.annotationsToContext(highlightedAnnotations, '#00BFFF');
-          }
-          if (selectionsUpdated) {
```

Comment:
These `if` statements were the cause of the odd behavior I was seeing before w/ the usage of `isEqual` in `annotationsMatch`.  We need to redraw all annotations every time we get here since we clear the canvas overly above.